### PR TITLE
[edit] API-Auth: Clarified instructions, added description

### PR DESCRIPTION
**Pull request by Catografix**
### Description

Clarified instructions, added description,, included ${account.namespace}, fixed hard-coded url. Updates PR #1232

### CODE REVIEW

File: articles/api-auth/asking-for-access-tokens.md [hunk 1]
```diff
@@ -1,46 +1,55 @@
...
+  "iss": "https://${account.namespace}/",
+  "sub": "APP_CLIENT_ID@clients",
+  "aud": "YOUR_API_IDENTIFIER",
   ...
   "scope": ""
 }
 ```
 
-> *NOTE:* If you executed the setup steps in the Auth0 Dashboard you will notice that the `scope` property is blank. At this point we don't provide support for custom scopes in the Auth0 Dashboard. You can still use this token to authorize access to the the parts of your API that don't require elevated permissions. If you would like to add scopes for your API you can follow [a few extra steps](/api-auth/adding-scopes).
+> **NOTE:** When you execute the setup steps in the Auth0 Dashboard as described in [API Authorization: Using the Auth0 Dashboard](/api-auth/using-the-auth0-dashboard), you will notice that the `scope` property is blank. Custom scopes are not supported in the Auth0 Dashboard at this time. You can still use this token to authorize access to the the parts of your API that do not require elevated permissions. If you would like to add scopes for your API you can follow [these steps](/api-auth/adding-scopes).
```
Comment (Catografix) [inline articles/api-auth/asking-for-access-tokens.md:31]: I don't see a scope property anywhere. Is this correct?

File: articles/api-auth/asking-for-access-tokens.md [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
-```
-https://{YOUR_NAMESPACE}/oauth/token
-```
+To ask Auth0 for tokens for any of your authorized client applications, perform a `POST` operation to the `https://${account.namespace}/oauth/token` endpoint with a payload in the following format:
 
-Payload:
 ```
 {
-  audience: "{YOUR_API_IDENTIFIER}",
+  audience: "YOUR_API_IDENTIFIER",
```
Comment (Catografix) [inline articles/api-auth/asking-for-access-tokens.md]: Removed curly brackets.

File: articles/api-auth/asking-for-access-tokens.md [hunk 3]
```diff
@@ -1,46 +1,55 @@
...
 ```
-The response will be a signed JWT token with
+The response will be a signed JWT token in this format:
 
 ```
 {
-  "iss": "https://{YOUR_NAMESPACE}/",
-  "sub": "{APP_CLIENT_ID}@clients",
-  "aud": "{YOUR_API_IDENTIFIER}",
+  "iss": "https://${account.namespace}/",
```
Comment (Catografix) [inline articles/api-auth/asking-for-access-tokens.md:23]: Replaced with dynamic text.

**Comment by Catografix** [comment]
Preview not working?

**Comment by ntotten** [comment]
@Catografix i triggered the preview manually. I'm not exactly sure whats going on here. I'll look into it to see why it isnt triggering automatically.

**Comment by Catografix** [comment]
@jerriep  Is the note about the blank scope property correct? I did not see this feild anywhere in the API setup pages at https://manage.auth0.com/#/apis/. Is this referring to a different setup page?

**Comment by Catografix** [comment]
Ready

**Pull request merged**